Makefile

```diff
@@ -6,8 +6,6 @@ PYTEST_CI_ARGS := --cov=. --cov-report=term-missing
 
 NPROCS = `nproc`
 
-CUDA_ARCH := "70"
```

Comment:
The default `CUDA_ARCH` setting was moved to CMakeLists.txt. This will allow `pip install -e .` to succeed without needing to set the CMake flags.